vscode/src/webview/docview.tsx

```diff
@@ -2,13 +2,433 @@
 // Licensed under the MIT License.
 
 import { Markdown } from "qsharp-lang/ux";
+import { useEffect, useState } from "preact/hooks";
 
-export function DocumentationView(props: { fragmentsToRender: string[] }) {
-  // Concatenate all documentation.
-  // The following adds an empty line and a horizontal line
-  // between documentation for different functions.
-  // We may consider filtering of fragments later.
-  const contentToRender = props.fragmentsToRender.join("<br/>\n\n---\n\n");
+export interface IDocFile {
+  filename: string;
+  metadata: string;
+  contents: string;
+}
+
+interface ItemDocs {
+  pkg: string;
+  module: string;
```

Comment:
Here and everywhere. Are we Ok using 'module' terminology?

Reply:
Changed to namespace